importScripts/sampleRecords/library.destination.json

```diff
@@ -21,7 +21,7 @@
       "ca_objects.PlaceOfPublication":"Mandurah",
       "ca_occurrences":[
         "Gee family - History",
-        "Truslove family - History."
+        "Truslove family"
```

Comment:
see row 4452 of https://docs.google.com/spreadsheets/d/1tQlbe0AZos9e8mVpxY_OaJMTT3Fq-rBPWkVbIuLwvY0/edit#gid=166863257